content/en/docs/instrumentation/python/exporters.md

```diff
@@ -0,0 +1,197 @@
+---
+title: "Exporters"
+weight: 4
+---
+
+In order to visualize and analyze your traces and metrics, you will need to
+export them to a backend.
+
+## Console exporter
+
+The console exporter is useful for development and debugging tasks, and is the
+simplest to set up.
+
+```python
+from opentelemetry import trace
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import BatchSpanProcessor, ConsoleSpanExporter
+from opentelemetry.sdk.resources import SERVICE_NAME, Resource
+
+# Service name is required for most backends
+resource = Resource(attributes={
+    "service.name": "your-service-name"
+})
+
+provider = TracerProvider(resource=resource)
+processor = BatchSpanProcessor(ConsoleSpanExporter())
+provider.add_span_processor(processor)
+trace.set_tracer_provider(provider)
+
+# Merrily go about tracing!
+```
+
+## OTLP endpoint or Collector
+
+To send data to an OTLP endpoint or the OpenTelemetry Collector, you'll want to
```

Comment:
Add a link to the collector: https://opentelemetry.io/docs/collector/getting-started/